### DISCUSSION TITLE
How to display table without components

### DISCUSSION THREAD

**Discussion by jcc5018** [rappasoft/laravel-livewire-tables#532] [Q&A] [answered]
Asked this in the docs repository, but I do not understand how to display the computed datatable. 

I do not use components as they do not make sense to me, so your examples in the docs do not really explain how to go from setting the column data, to actually displaying the data on a page. If your package uses the components and handles it without needing modification from me, I can keep it as is. But What blade file do I need to create? or include in order to see the computed settings? 

in your instructions you say : " I don't recommend you publishing the views unless you really need to change them, and if so, only keep the ones you are changing. Let the package display the rest. The views change quite often and you will miss out on new features or have unforeseeable issues."

How do I access the created table?

**Comment by rappasoft** [comment, marked as answer]
It doesn't seem like you have a grasp on the [basics of livewire](https://laravel-livewire.com/docs/2.x/rendering-components#inline-components), which would be a prerequisite to using this package.

You don't need to make any views, the PHP class is enough to do everything and you just call it with Livewire to render. Publishing the views is if you don't like the default way the package looks but isn't recommended. 

Generally you call a Livewire component based on where the component class lives in relation to `app/Http/Livewire`. So if you have a UsersTable Component in that directory, you'd render it with:

```html
<livewire:users-table />
```

**Reply by jcc5018** [reply]
Gotcha, eah I am still trying to get a grasp of livewire, and Laravel in general. I managed to create my own Data table, but I'm not sure about the whole column generation methods.  But I will give it a try now that I figured out how to make it work. 

I'm used to the controllers, calling a view, or in the case of livewire, having that render method, and a blade file where I could  edit things.  I'll see if your package can handle what i need it to do. Thanks.

**Reply by fahmij8** [reply]
@rappasoft Tbh, this one should be on the documentation. I am so confused about how to render the components as there's no blade file whatsoever. Anyway thanks!
